superset-frontend/src/dashboard/components/SaveModal.tsx

```diff
@@ -69,25 +87,25 @@ class SaveModal extends React.PureComponent {
     this.onSave = this.props.onSave.bind(this);
   }
 
-  setModalRef(ref) {
+  setModalRef(ref: ModalTrigger | null) {
     this.modal = ref;
   }
 
-  toggleDuplicateSlices() {
+  toggleDuplicateSlices(val?: boolean | undefined): void {
```

Comment:
you don't need the `| undefined` if you already have a `?:`

Although, I wonder why you added an argument here since it seems like the function doesn't actually use it? Is that because you pass it into a component that expects a function with this arg?